### PR TITLE
LPS-169211  Add support for Freemarker script in the notification text

### PR DESCRIPTION
**Pull request by danvb112**
Related Issues: 

https://issues.liferay.com/browse/LPS-169045
https://issues.liferay.com/browse/LPS-169211
https://issues.liferay.com/browse/LPS-168333
https://issues.liferay.com/browse/LPS-168336

### CODE REVIEW

**Comment by danvb112** [comment]
ci:test:sf

**Comment by danvb112** [comment]
ci:test:relevant

**Comment by gabrielwas** [comment]
Waiting for the merge of https://issues.liferay.com/browse/LPS-168336

**Comment by danvb112** [comment]
ci:test:sf

File: modules/apps/object/object-js-components-web/src/main/resources/META-INF/resources/components/CodeEditor/CodeEditorLocalized.tsx [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+	}));
+
+export function CodeEditorLocalized({
+	CustomSidebarContent,
+	ariaLabels = {
+		default: 'Default',
+		openLocalizations: 'Open Localizations',
+		translated: 'Translated',
+		untranslated: 'Untranslated',
+	},
```
Comment (cgmonte) [inline modules/apps/object/object-js-components-web/src/main/resources/META-INF/resources/components/CodeEditor/CodeEditorLocalized.tsx:66]: Why not translating ariaLabels? Like:
`ariaLabels = {default: Liferay.language.get('default') ...},`

**Review by cgmonte — CHANGES_REQUESTED** [review]
Hey @danvb112! Great job on this PR :) Please review my comments. The one about translating aria labels is a low effort change which delivers great value for people with disabilities in non english speaking countries. A second onde just asks to open a technical debit. Thanks

**Comment by danvb112** [comment]
ci:test:sf

File: modules/apps/object/object-js-components-web/src/main/resources/META-INF/resources/components/CodeEditor/CodeEditorLocalized.tsx (hunk 1 above)
Comment (danvb112) [reply modules/apps/object/object-js-components-web/src/main/resources/META-INF/resources/components/CodeEditor/CodeEditorLocalized.tsx:66]: Thanks for noticing that @cgmonte, I already made your suggestion :smile:

**Comment by danvb112** [comment]
ci:test:sf

**Comment by danvb112** [comment]
sent to: https://github.com/brianchandotcom/liferay-portal/pull/126804

**Pull request closed without merging**